samples/cpp/stitching_detailed.cpp

```diff
@@ -326,8 +328,10 @@ static int parseCmdArgs(int argc, char** argv)
                 return -1;
             }
             i++;
-
-            timelapse_range = atoi(argv[i + 1]);
+        }
+        else if (string(argv[i]) == "--rangewidth")
+        {
+            range_width = atoi(argv[i + 1]);
```

Comment:
My suggestion is to add `timelapse = true;` somewhere here in case of renamed parameter.